### PR TITLE
rakudo-star: updated to 2016.03

### PR DESCRIPTION
**Pull request by xaduha**

### CODE REVIEW

**Comment by Vaelatern** [comment]
Please have one commit for each version bump.

File: srcpkgs/rakudo/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 pkgname=rakudo
 only_for_archs="i686 x86_64"
-version=2016.02
+version=2016.03
 revision=1
 build_style=configure
 configure_script="perl Configure.pl"
 configure_args="--prefix=/usr --backends=moar"
 hostmakedepends="perl MoarVM nqp"
-makedepends="${hostmakedepends} libatomic_ops-devel libffi-devel libuv-devel"
```
Comment (chneukirchen) [inline srcpkgs/rakudo/template]: why this again?

Comment (xaduha) [inline srcpkgs/rakudo/template]: Rakudo really depends on MoarVM and nqp, MoarVM has those dependencies itself.

Comment (Vaelatern) [inline srcpkgs/rakudo/template]: No.

File: srcpkgs/rakudo-star/template [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=rakudo-star
 only_for_archs="i686 x86_64"
-version=2016.02
+version=2016.03
 revision=1
 hostmakedepends="panda"
 makedepends="${hostmakedepends}"
-depends="panda>=2015.12"
+depends="panda>=2016.02"
 short_desc="Rakudo is an implementation of Perl 6 language"
```
Comment (Vaelatern) [inline srcpkgs/rakudo-star/template]: Please have a better description for what this package actually does.

**Comment by Vaelatern** [comment]
@chneukirchen This looks good to me...

**Pull request merged**